apollographql/router-defer-e2e-tests
# chore: test mutation with defer

To test a mutation with `@defer`:

- clone repo, pull down `mutations-test` branch
- `cd CRA-demo && npm run router:up`
- `npm i && npm start`
- browser should open to localhost:3000, navigate to https://localhost:3000/purchase
- click button, inspect response in network tab

NB: users schema came from @lleadbet's https://github.com/lleadbet/graphql-galaxy-demo
